### ISSUE TITLE
OpenNebula generates zeroed virtual_router_id

### ISSUE THREAD

**Issue by ospalax** [OpenNebula/one#4220]
**Description**
When no `VROUTER_KEEPALIVED_ID` is provided OpenNebula generates it based on the internal id and by simple binary operation `and (&)` which can result in zero if internal id is zero (or has lower 8 bits zeroed) - code reference:
https://github.com/OpenNebula/one/blob/7df413faeba1f890f05a980617ef6919b72537d3/src/vrouter/VirtualRouter.cc#L489-L494

This `virtual_router_id` is used inside keepalived's vrrp instance and keepalived [documentation](https://www.keepalived.org/manpage.html) says this about it:
```
           # arbitrary unique number from 1 to 255
           # used to differentiate multiple instances of vrrpd
           # running on the same NIC (and hence same socket).
           virtual_router_id 51
```
The result (with `0` as `virtual_router_id`) can result in this:
```
b 19 15:32:39 localhost daemon.info Keepalived[3255]: Reloading ...
Feb 19 15:32:39 localhost daemon.info Keepalived[3255]: Opening file '/etc/keepalived/keepalived.conf'.
Feb 19 15:32:39 localhost local1.info Keepalived_vrrp[3266]: Reloading
Feb 19 15:32:39 localhost local1.info Keepalived_vrrp[3266]: Opening file '/etc/keepalived/keepalived.conf'.
Feb 19 15:32:39 localhost local1.info Keepalived_vrrp[3266]: (Line 20) number '0' outside range [1, 255]
Feb 19 15:32:39 localhost local1.info Keepalived_vrrp[3266]: (Line 20) (ETH0): VRID '0' not valid - must be between 1 & 255
Feb 19 15:32:39 localhost local1.info Keepalived_vrrp[3266]: (Line 32) number '0' outside range [1, 255]
Feb 19 15:32:39 localhost local1.info Keepalived_vrrp[3266]: (Line 32) (ETH1): VRID '0' not valid - must be between 1 & 255
Feb 19 15:32:39 localhost local1.info Keepalived_vrrp[3266]: (Line 44) number '0' outside range [1, 255]
Feb 19 15:32:39 localhost local1.info Keepalived_vrrp[3266]: (Line 44) (ETH2): VRID '0' not valid - must be between 1 & 255
Feb 19 15:32:39 localhost local1.info Keepalived_vrrp[3266]: (ETH0) the virtual router id must be set
Feb 19 15:32:39 localhost local1.info Keepalived_vrrp[3266]: Stopped
Feb 19 15:32:39 localhost daemon.info Keepalived[3255]: Keepalived_vrrp exited with permanent error CONFIG. Terminating
Feb 19 15:32:39 localhost daemon.info Keepalived[3255]: Stopped Keepalived v2.0.19 (11/24,2019), git commit v20191114-652-g038556d644
```

**To Reproduce**
Start completely fresh OpenNebula and create bare vrouter context with just `VROUTER=YES` and do not set `VROUTER_KEEPALIVED_ID` - if this vrouter was first it may have internal id of zero - or spin many of them until vrouter ids cross 255 (256 is binary one followed by 8 zeroes).

**Expected behavior**
Generated vrouter ids should be in a range: 1-255

**Additional context**
https://github.com/OpenNebula/one/blob/7df413faeba1f890f05a980617ef6919b72537d3/src/vrouter/VirtualRouter.cc#L489-L494

## Progress Status
- [ ] Branch created 
- [ ] Code committed to development branch
- [ ] Testing - QA
- [ ] Documentation 
- [ ] Release notes - resolved issues, compatibility, known issues
- [ ] Code committed to upstream release/hotfix branches
- [ ] Documentation committed to upstream release/hotfix branches

**Comment by al3xhh** [comment]
PRs to merge in master:

- code: https://github.com/OpenNebula/one/pull/4620
- docs: https://github.com/OpenNebula/docs/pull/1089
- tests: https://github.com/OpenNebula/development/pull/943

**Comment by ospalax** [comment]
Fixed by [#4220](https://github.com/OpenNebula/one/pull/4620)
